trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
```

Comment:
- [ ] **Required:** This method should return only the first 9 characters if a name is longer than that. It does work as required. Consider the code snippet below.

```
def correct_name
   @nameable.correct_name[0..9] 
end
```